static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -19,24 +19,24 @@ import fastdom from 'fastdom';
 import raven from 'lib/raven';
 import userPrefs from 'common/modules/user-prefs';
 import images from 'common/modules/ui/images';
-import storage from 'lib/storage';
+import { localStorage } from 'lib/storage';
```

Comment:
yeah there are a few reasons it may fail (full, private mode etc), but it really only need a try/catch i think?